File: Python/Prodi.py
# library
from Course import Course

# membuat kelas Prodi 
class Prodi():
    # atribut privat kelas Prodi
    __nama_prodi = ""
    __kode = ""
    __course = Course()

    # konstruktor dengan parameter
    def __init__(self, nama_prodi = "", kode = "", course = Course()):
        #set nilainya
        self.__nama_prodi = nama_prodi
        self.__kode = kode
        self.__course = course
    
    # tetapkan nama_prodi
    def setNama_Prodi(self, nama_prodi):
        self.__nama_prodi = nama_prodi
    
    # get nama_prodi
    def getNama_Prodi(self):
        return self.__nama_prodi
    
    # tetapkan kode
    def setKode(self, kode):
        self.__kode = kode

    # get kode
    def getKode(self):
        return self.__kode
    
    # tetapkan course
    def setCourse(self, course):
        self.__course = course

    # get course
    def getCourse(self):
        return self.__course

File: CPP/Prodi.cpp
// library
#include <string>
#include "Course.cpp"

using namespace std;

// membuat kelas Prodi
class Prodi{
    // atribut yang diperlukan
    private:
        string nama_prodi;
        string kode;
        Course course;
    
    public:
        // konstruktor kosong
        Prodi() {
        }

        // konstruktor dengan parameter
        Prodi(string nama_prodi, string kode, Course course) {
            // set nilainya
            this->nama_prodi = nama_prodi;
            this->kode = kode;
            this->course = course;
        }

        // tetapkan kode
        void setKode(string kode) {
            this->kode = kode;
        }

        // get kode
        string getKode() {
            return this->kode;
        }

         // tetapkan nama_prodi
        void setNama_Prodi(string nama_prodi) {
            this->nama_prodi = nama_prodi;
        }

        // get nama_prodi
        string getNama_Prodi() {
            return this->nama_prodi;
        }

         // tetapkan course
        void setCourse(Course course) {
            this->course = course;
        }

        // get course
        Course getCourse() {
            return this->course;
        }
        
        // deconstruct
        ~Prodi() {
        }
};